relay-server/src/utils/dynamic_sampling.rs

```diff
@@ -15,79 +15,63 @@ use crate::envelope::{Envelope, ItemType};
 /// Checks whether an event should be kept or removed by dynamic sampling.
```

Comment:
_nit_: We should update the doc comment and mention explicitly that this runs both trace and event rules.